src/index.css

```diff
@@ -82,3 +82,7 @@ body {
 .retro-text::before {
   -webkit-text-stroke: 4px transparent !important;
 }
+
+.outlined-button {
```

Comment:
Maybe `secondary-button`?

Reply:
I think that outlined button would be a better choice, because we're passing outlined prop into our `Button.tsx` component for making button with green background. And, for consistency, I think that we should name this style outlined as well.